lambdas/utils/lambda_response.py

```diff
@@ -1,5 +1,5 @@
 class ApiGatewayResponse:
-    def __init__(self, status_code, body, methods) -> None:
+    def __init__(self, status_code: object, body: object, methods: object) -> None:
```

Comment:
status code should be 'int', body is 'str', and methods can be 'str' or a list

Reply:
Weirdly, I didn't change this..... This might be some automation, or it might have been pulled in from elsewhere
I'll make the change and hope it doesn't break......